plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/TestMongoIntegrationSmokeTest.java

```diff
@@ -191,6 +193,21 @@ public void testSkipUnknownTypes()
         assertQueryReturnsEmptyResult("SHOW COLUMNS FROM test.tmp_guess_schema2");
     }
 
+    @Test
+    public void testDBRef()
+    {
+        Document document = Document.parse("{\"_id\":ObjectId(\"5126bbf64aed4daf9e2ab771\"),\"col1\":\"foo\"}");
+
+        ObjectId objectId = new ObjectId("5126bc054aed4daf9e2ab772");
+        DBRef dbRef = new DBRef("test", "creators", objectId);
+        document.append("creator", dbRef);
+
+        client.getDatabase("test").getCollection("test_dbref").insertOne(document);
+        assertQuery("SELECT count(*) FROM test.test_dbref WHERE creator IS NOT NULL", "SELECT 1");
+        assertQuery("SELECT creator.databaseName, creator.collectionName, cast(creator.id as varchar) FROM test.test_dbref WHERE creator IS NOT NULL", "SELECT 'test', 'creators', '5126bc054aed4daf9e2ab772'");
+        assertQuery("SELECT typeof(creator) FROM test.test_dbref", "SELECT 'row(databaseName varchar, collectionName varchar, id ObjectId)'");
+    }
```

Comment:
Please fix like below to improve readability. (Add a new line before `assertQuery`, uppercase some keywords, wrap arguments of `assertQuery`)
```java
        client.getDatabase("test").getCollection("test_dbref").insertOne(document);

        assertQuery(
                "SELECT creator.databaseName, creator.collectionName, CAST(creator.id AS VARCHAR) FROM test.test_dbref",
                "SELECT 'test', 'creators', '5126bc054aed4daf9e2ab772'");
        assertQuery(
                "SELECT typeof(creator) FROM test.test_dbref",
                "SELECT 'row(databaseName varchar, collectionName varchar, id ObjectId)'");
```

Reply:
done